package.json

```diff
@@ -16,7 +16,9 @@
       "require": "./dist/index.js",
       "import": "./dist/index.mjs",
       "types": "./dist/index.d.ts"
-    }
+    },
+    "./cypress": "./dist/cypress-api/index.js",
+    "./cypress/support": "./dist/cypress-api/support.js"
```

Comment:
I added these as entry points so we can install them separately from Playwright (and in a fashion similar to how [other Cypress plugins](https://github.com/cypress-io/code-coverage#install) do it)